src/hearings/resolvers/judicial-user-search-response-resolver.resolve.ts

```diff
@@ -33,9 +33,11 @@ export class JudicialUserSearchResponseResolver implements Resolve<JudicialUserM
     return this.hearingStore.pipe(select(fromHearingStore.getHearingRequest)).pipe(
       map(hearingRequest => {
         let panelMemberIds: string[] = [];
+        let hearingJudgeIds: string[] = [];
         if (hearingRequest.hearingRequestMainModel && hearingRequest.hearingRequestMainModel.hearingResponse && hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule) {
-          panelMemberIds = hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule.panelMemberIds || [];
-          panelMemberIds = [...panelMemberIds, hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule.hearingJudgeId];
+          panelMemberIds = hearingRequest.hearingRequestMainModel.hearingResponse.hearingDaySchedule.map(member => member.panelMemberId).filter(memberId => memberId) || [];
```

Comment:
can this panelmember return more than 1 item as all other places we have changed into non-array ?

Reply:
hearingDaySchedule is an array. So, getting all paelMenberId and judgeId from hearingDaySchedule[]